File: 240519/약수가 세 개인 수/numbers-with-three-divisors.py
start, end = input().split()
start = int(start); end = int(end)

cnt = 0 

for i in range(start, end+1):

    sum_ = 0
    for j in range(1, i+1):
        if i%j == 0:
            sum_ += 1
    if sum_ == 3:
        cnt += 1
        
print(cnt)

File: 241223/약수가 세 개인 수/numbers-with-three-divisors.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int start, end, count = 0;
    cin >> start >> end;

    for (; start<=end; start++) {
        int cnt = 0;
        for (int i=1; i<=start; i++) {
            if (start%i == 0) {
                cnt++;
            }
        }
        if (cnt == 3) {
            count++;
        }
    }
    cout << count;
    return 0;
}